Resources/Prototypes/Entities/Objects/Consumable/Smokeables/base_smokeables.yml

```
@@ -1,86 +1,87 @@
+  components:
+  - type: Reactive
+    groups:
+      Extinguish: [ Touch ]
+  - type: ExtinguishableSetCollisionWake
+  - type: Smokable
+  - type: Sprite
+  - type: Appearance
+  - type: BurnStateVisuals
+  - type: Tag
+    tags:
+    - Trash
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)